modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/misc/GithubApi2.kt

```diff
@@ -110,38 +114,42 @@ class GithubApi2 @Inject constructor(
         get() {
             val callable = Callable<String> {
                 var result = "0"
```

Comment:
While you're at it, I strongly suspect this is why we sometimes find a `0` in `kobalt-wrapper.properties` when something goes wrong. A more sensible default would be the same version that was in the original file.

But if you're going to fix this, do it in a separate PR.